src/ServerTrait.php

```diff
@@ -29,4 +29,15 @@ private function stopWebServer(): void
     {
         PantherTestCase::stopWebServer();
     }
+
+    private function pause($message): void
+    {
+        if (PantherTestCase::isWebServerStarted()
+            && \in_array('--debug', $_SERVER['argv'], true)
+            && $_SERVER['PANTHER_NO_HEADLESS'] ?? false
+        ) {
+            echo "$message\n\nPress enter to continue...";
+            fgets(STDIN);
```

Comment:
You could create an `@internal` 'fgetsMock` property, and call it if defined instead of `fgets` or something similar.

Reply:
I'm sorry, I don't really see what you mean...

Reply:
ServerTrait is already internal. Could we use a public `testing` property to make simpler the code ?

Reply:
LGTM

Reply:
Done.